hw1/02-palindrome.css

```diff
@@ -0,0 +1,42 @@
+body {
+    font-family: Arial, sans-serif;
+    background-color: #5250f2;
+}
+
+.container {
```

Comment:
I would recommend using Bootstrap as most, if not all of this can be quickly written up in it. Tailwind, this can for sure be written in it.